File: perfect_sound_mobile/lib/models/Rating/Rating.dart
class Rating {
  int? userId;
  int? songAndPodcastId;
  double? ratingValue;

  Rating({this.userId, this.songAndPodcastId, this.ratingValue});

  Rating.fromJson(Map<String, dynamic> json) {
    userId = json['userId'];
    songAndPodcastId = json['songAndPodcastId'];
    if(json['ratingValue'].runtimeType==int){
      int value=json['ratingValue'];
      ratingValue=value.toDouble();
    }
    else
      ratingValue = json['ratingValue'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['userId'] = this.userId;
    data['songAndPodcastId'] = this.songAndPodcastId;
    data['ratingValue'] = this.ratingValue;
    return data;
  }
}

File: PerfectSound/PerfectSound.Model/Model/Rating.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace PerfectSound.Model.Model
{
    public class Rating
    {
        public int RatingId { get; set; }

        public int UserId { get; set; }
        public int SongAndPodcastId { get; set; }
        public double RatingValue { get; set; }
    }
}
